modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+
+export const test = mergeTests(dataSetsPageTest, loginTest);
```

Comment:
You might need to include the `featureFlagsTest` fixture. Otherwise, the Data Set menu option won't be available.

Reply:
Hmm, the test passes without it. Is it really necessary? Do you think it could be causing the SF error that CI is reporting?

Reply:
It will pass locally if you have defined the FF in your `portal-ext.properties`, but CI will complain

Reply:
SF errors seems to be related with yesterday changes the `post.spec.ts` [file](https://github.com/liferay-frontend/liferay-portal/commit/69195cc2b1c9f64b4c92b1dc7067d4feeb10089e#diff-5de95d9e539758f3e0e4a6c400d2e6e7acfc63968db8caca24bde39486a99b9fL80). They forgot to change this last `basicAPIApplication` reference

Reply:
Just update with latest master changes. There will be some changes to the creation of a DataSet :pray:

Reply:
👍 ty @juanjofgliferay